src/entities/inputValidator/liquidityBootstrapping/inputValidatorLiquidityBootstrapping.ts

```diff
@@ -0,0 +1,55 @@
+import { AddLiquidityInput } from '@/entities/addLiquidity/types';
+import { InputValidatorBase } from '../inputValidatorBase';
+import { PoolState, PoolStateWithBalances } from '../../types';
+import {
+    RemoveLiquidityInput,
+    RemoveLiquidityRecoveryInput,
+} from '../../removeLiquidity/types';
+import { CreateLiquidityBoostrappingPoolInput } from '../../createPool/types';
+
+export class InputValidatorLiquidityBootstrapping extends InputValidatorBase {
+    // TODO
+    validateAddLiquidity(
+        _addLiquidityInput: AddLiquidityInput,
+        _poolState: PoolState,
+    ): void {}
+    // TODO
+    validateRemoveLiquidity(
+        _removeLiquidityInput: RemoveLiquidityInput,
+        _poolState: PoolState,
+    ): void {}
+    // TODO
+    validateRemoveLiquidityRecovery(
+        _removeLiquidityRecoveryInput: RemoveLiquidityRecoveryInput,
+        _poolStateWithBalances: PoolStateWithBalances,
+    ): void {}
+    validateCreatePool(input: CreateLiquidityBoostrappingPoolInput): void {
+        // start weights
+        const startWeightsSum =
+            input.lbpParams.projectTokenStartWeight +
+            input.lbpParams.reserveTokenStartWeight;
+        if (startWeightsSum !== BigInt(1e18)) {
+            throw new Error('Start weights must sum to 100');
+        }
+        // end weights
+        const endWeightsSum =
+            input.lbpParams.projectTokenEndWeight +
+            input.lbpParams.reserveTokenEndWeight;
+        if (endWeightsSum !== BigInt(1e18)) {
+            throw new Error('End weights must sum to 100');
+        }
+        // validate start and end times
+        if (input.lbpParams.startTime >= input.lbpParams.endTime) {
+            throw new Error('Start time must be before end time');
+        }
+        // cannot be in the past - technically allowed on the sc side.
+        // will simply move startTime to now.
+        if (input.lbpParams.startTime < BigInt(Math.floor(Date.now() / 1000))) {
+            throw new Error('Start time must be in the future');
+        }
+        // tokens cannot be the same
+        if (input.lbpParams.projectToken === input.lbpParams.reserveToken) {
```

Comment:
Please use `isSameAddress` helper to avoid issues with case sensitive comparisons